src/microbe_stage/PatchMap.cs

```diff
@@ -215,6 +215,7 @@ public List<Species> RemoveExtinctSpecies(bool playerCantGoExtinct = false)
 
         foreach (var patch in Patches)
         {
+            // Looks like player species is selected iff not playercantgoextinct...
```

Comment:
```suggestion
            // Player species can only go extinct when playerCantGoExtinct is true
```

Reply:
This was a leftover note to self, because it seemed there was a bug lying here: player species should go extinct only when playerCantGoExtinct is False...

Reply:
I reworded this so that it makes sense in general. I think this can be kept as an explanation to future programmers to see.

Reply:
Yeah, but this explanation (matching the code) is contradictory, so this may be confusing. Though, I guess It makes the issue more visible, so we may in fact keep your wording (which can be changed after correction).

Reply:
Oh, did I accidentally write the opposite of what I should have...

Reply:
After looking at the code carefully, it's starting to look like the logical condition is actually the wrong way around in the code here? If you don't mind, could you make a separate PR from flipping that condition the right way around so that we can discuss this there?

Reply:
Yes, I think the issue is indeed that it should be !PlayerCantGoExtinct in the patch. I'll open a PR next time I boot on windows, should be either tonight or tomorrow. Made myself a memo for this